### PR TITLE
[4.0] bye bye xref

### PR DESCRIPTION
**Pull request by brianteeman**
WIP

Pull Request for Issue #25272 

Documents it, Explain it or just kill it

No point having code no one knows what to do with it.

We have enough fields to confuse people without two extra fields

### CODE REVIEW

File: components/com_content/Model/ArticleModel.php [hunk 1]
```diff
@@ -104,7 +104,7 @@ public function getItem($pk = null)
 							'CASE WHEN a.modified = ' . $db->quote($db->getNullDate()) . ' THEN a.created ELSE a.modified END as modified, ' .
 							'a.modified_by, a.checked_out, a.checked_out_time, a.publish_up, a.publish_down, ' .
 							'a.images, a.urls, a.attribs, a.version, a.ordering, ' .
-							'a.metakey, a.metadesc, a.access, a.hits, a.metadata, a.featured, a.language, a.xreference'
+							'a.metakey, a.metadesc, a.access, a.hits, a.metadata, a.featured, a.language
```
Comment (Quy) [inline components/com_content/Model/ArticleModel.php]:
```suggestion
							'a.metakey, a.metadesc, a.access, a.hits, a.metadata, a.featured, a.language'
```

File: administrator/components/com_content/forms/article.xml [hunk 2]
```diff
@@ -654,14 +654,6 @@
 		</fieldset>
 	</fields>
 
-	<field
-		name="xreference"
-		type="text"
-		label="JFIELD_KEY_REFERENCE_LABEL"
-		description="JFIELD_KEY_REFERENCE_DESC"
```
Comment (Quy) [inline administrator/components/com_content/forms/article.xml]: Remove language strings.

File: administrator/components/com_contact/sql/install.mysql.utf8.sql [hunk 3]
```diff
@@ -54,5 +53,4 @@ CREATE TABLE IF NOT EXISTS `#__contact_details` (
   KEY `idx_createdby` (`created_by`),
   KEY `idx_featured_catid` (`featured`,`catid`),
   KEY `idx_language` (`language`),
```
Comment (Quy) [inline administrator/components/com_contact/sql/install.mysql.utf8.sql]:
```suggestion
  KEY `idx_language` (`language`)
```

File: administrator/components/com_newsfeeds/sql/install.mysql.utf8.sql [hunk 4]
```diff
@@ -40,5 +39,4 @@ CREATE TABLE IF NOT EXISTS `#__newsfeeds` (
   KEY `idx_catid` (`catid`),
   KEY `idx_createdby` (`created_by`),
   KEY `idx_language` (`language`),
```
Comment (Quy) [inline administrator/components/com_newsfeeds/sql/install.mysql.utf8.sql]:
```suggestion
  KEY `idx_language` (`language`)
```

**Comment by wilsonge** [comment]
It's unused kill it

**Comment by Hackwar** [comment]
Bye bye

**Comment by brianteeman** [comment]
conflicts resolved

**Comment by wilsonge** [comment]
Something went wrong with conflict resolution :/

**Comment by brianteeman** [comment]
grrh

**Comment by brianteeman** [comment]
See #25549

**Pull request closed without merging**